Kernel/kprintf.cpp

```diff
@@ -70,7 +70,13 @@ static void console_putch(char*&, char ch)
 {
     if (serial_debug)
         serial_putch(ch);
-    Console::the().put_char(ch);
+
+    // It would be bad to reach the assert in Console()::the() and do a stack overflow
+
+    if (Console::is_initialized())
```

Comment:
Please don't put a newline before `{`. Just use `clang-format` if you are unsure about the coding style.